native/src/main/java/io/ballerina/stdlib/graphql/runtime/utils/Utils.java

```diff
@@ -76,4 +80,19 @@ public static boolean isContext(Type type) {
                 type.getPackage().getName().equals(ModuleUtils.getModule().getName()) &&
                 type.getName().equals(CONTEXT_OBJECT);
     }
+
+    public static boolean isFileUpload(Type type) {
+        if (type.getTag() == TypeTags.ARRAY_TAG) {
+            return isFileUpload(((ArrayType) type).getElementType());
+        }
+        if (type.getPackage() == null) {
+            return false;
+        }
+        if (type.getPackage().getOrg() == null || type.getPackage().getName() == null) {
+            return false;
+        }
+        return type.getPackage().getOrg().equals(ModuleUtils.getModule().getOrg()) &&
+                type.getPackage().getName().equals(ModuleUtils.getModule().getName()) &&
+                type.getName().equals(FILE_UPLOAD);
+    }
```

Comment:
Shall we move this code to a separate function (maybe `isGraphqlModule`), and re-use it? This is otherwise duplicated in `isContext()` method, right?

We can first check `isGraphqlModule()` and then inside this method, check the type name.